addons/web/static/src/owl2_compatibility/app.js

```diff
@@ -0,0 +1,95 @@
+(function () {
```

Comment:
move this code in test folder, and no need to write on App, it can simply be imported in test utils